hummingbot/client/config/config_helpers.py

```diff
@@ -186,7 +186,7 @@ def get_erc20_token_addresses() -> Dict[str, List]:
                 override_list.append(token)
 
             if len(override_list) > 0:
-                print(f"Applied address overrides from /conf/erc20_tokens.json to symbols {override_list}.")
+                print(f"Applied address overrides from {TOKEN_ADDRESSES_FILE_PATH} to symbols {override_list}.")
```

Comment:
It'd be better to log this, st. we have the message printed on the screen as well as logged into a file (for later analysis if needed).